### PR TITLE
Beers2

### PR DESCRIPTION
**Pull request by mattphoto**

### CODE REVIEW

File: Procfile [hunk 1]
```diff
@@ -1 +1 @@
-web: node keystone.js
+web: nodemon keystone.js
```
Comment (lizTheDeveloper) [inline Procfile]: Do not commit this file. Remove it from the commit, or add a new commit where you change this back.

File: templates/views/beerProfile.html [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+{% block intro %}
+
+{% endblock %}
+
+{% block content %}
+
+</script>
+<div class="bodyToLoad">
+			<div class="row">
```
Comment (lizTheDeveloper) [inline templates/views/beerProfile.html]: Fix these tabs while you're fixing things

File: templates/views/beerProfile.html [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+			</div>
+		</div>
+
+		<br />
+		<br />
+		<br />
+
+
+		<div class="row">
+			<div class="col-md-6">
```
Comment (lizTheDeveloper) [inline templates/views/beerProfile.html]: Your tab width is _insane_ here, let's keep our tab width to 4 spaces

File: templates/views/beers.html [hunk 4]
```diff
@@ -1,23 +1,88 @@
...
+    {% if loop.index0%6 == 0 %}
+    <div class="col-md-12" align="center">
+      {% endif %}
+        <a href="/beerProfile/{{beer.slug}}">
+        {% if beer.beerImage.url != nil %}
+        <div class="col-md-2" align="center">
+          <img src="{{beer.beerImage.url}}" class="img-thumbnail">
+          <br>
+          {{beer.beerName}}
+        </a>
```
Comment (lizTheDeveloper) [inline templates/views/beers.html]: You need to properly nest your tags, they're like <a> <div> </a> </div> and that's not how HTML works.

File: templates/views/beers.html [hunk 5]
```diff
@@ -1,23 +1,88 @@
...
+      </div>
+      {% else %}
+      <div class="col-md-2" align="center">
+        <a href="/beerProfile/{{beer.slug}}">
```
Comment (lizTheDeveloper) [inline templates/views/beers.html:65]: This one is right, make the above one look like this one.

File: templates/views/beers.html [hunk 6]
```diff
@@ -1,23 +1,88 @@
...
-            <p>All beers</p>
-        </div>
+  <div class="col-md-12" align="center">
+    <div class="container">
+      <p>Featured beers</p>
+      <button class="btn btn-primary">Try these beers</button>
     </div>
-{% endblock %}
\ No newline at end of file
+  </div>
```
Comment (lizTheDeveloper) [inline templates/views/beers.html]: Your closing divs should be in the block not outside it. Nothing outside of blocks gets actually put on the page.

File: models/beers.js [hunk 7]
```diff
@@ -1,36 +0,0 @@
...
-    {
-      profileTitle: { type: String },
-      profileAuthor: { type: Types.Name },
-      profileBody: { type: Types.Textarea },
-      flavorProfile: { type: Types.Select, options: 'Malty, Earthy, Aromatic, Sweet, Rich, Delicious', default: 'Delicious', index: true },
-      profileDate: { type: Types.Date, default: Date.now }
-    }
-);
-
-Beers.relationship({ ref: 'Releases', path: 'Releases', refPath: 'beersOfRelease'});
```
Comment (lizTheDeveloper) [inline models/beers.js:33]: Add this line to the Beers.js file

Comment (mattphoto) [inline models/beers.js:33]: added

File: routes/views/beers.js [hunk 8]
```diff
@@ -1,14 +1,44 @@
...
-	view.render('beers');
-	
-};
+  
+  var view = new keystone.View(req,res),
+  locals = res.locals;
+
+  locals.section = 'beers';
+  locals.filters = {
+    featuredBeers: []
```
Comment (lizTheDeveloper) [inline routes/views/beers.js]: Should be in the data, not the filters

File: templates/views/beers.html (hunk 6 above)
Comment (lizTheDeveloper) [inline templates/views/beers.html]: These won't be rendered

**Pull request merged**